tests/test_E3_indicator_calculation.py

```diff
@@ -1,5 +1,6 @@
 import pandas as pd
 import pytest
+import numpy as np
 
```

Comment:
Nothing wrong here, but the .py file appears all grey (no code highlighting) in github. Any ideas to why?

Reply:
hmm where does it appear grey for you? 
[here](https://github.com/rl-institut/multi-vector-simulator/blob/feature/co2_constraint_and_kpi/tests/test_E3_indicator_calculation.py) and in the files changed tab I see the code highlighting.

Reply:
Weird...